### PR TITLE
[14.0][MIG] partner_contact_job_position: Migration to 14.0

### PR DESCRIPTION
**Pull request by newtratip**
Standard Migration

### CODE REVIEW

**Review by tonygalmiche — APPROVED** [review]
LGTM : Code and functional review

File: partner_contact_job_position/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+To use this module, you need to:
+
+* Go to any partner's form.
+* Use the new *Categorized job position* field.
```
Comment (emagdalenaC2i) [inline partner_contact_job_position/readme/USAGE.rst:4]: Please, add a few lines describing the option to add/see/edit Job positions in Contacts --> Configuration --> Job Positions

Comment (newtratip) [reply partner_contact_job_position/readme/USAGE.rst:4]: I add how to config `job position` in configuration section. Is this same above text ?
Selection_400

Comment (emagdalenaC2i) [reply partner_contact_job_position/readme/USAGE.rst:4]: Ok, sorry I didn't notice there is in the configuration file

**Review by emagdalenaC2i — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]

**Pull request merged**